relay-base-schema/src/data_category.rs

```diff
@@ -65,6 +65,10 @@ pub enum DataCategory {
     UserReportV2 = 14,
     /// Metric buckets.
     MetricBucket = 15,
+    /// SpanIndexed
+    ///
+    /// This is the category for spans we store in full.
+    SpanIndexed = 16,
```

Comment:
As discussed: We need to release librelay with this new category and then bump it in sentry and snuba before we start producing outcomes with this ID.

Reply:
https://github.com/getsentry/relay/pull/2937